### PR TITLE
Provide IRelationBrokenEvent to be able to distinguish the event when it is broken

### PR DESCRIPTION
**Pull request by vangheem**

### CODE REVIEW

File: src/z3c/relationfield/event.py [hunk 1]
```diff
@@ -2,18 +2,27 @@
...
 from zope.interface import providedBy
 from zope.lifecycleevent import ObjectModifiedEvent
 from zope.schema import getFields
 
 
+@implementer(IRelationBrokenEvent)
+class RelationBrokenEvent(ObjectModifiedEvent):
+    '''
+    Event that is triggered when relation is broken.
+    '''
```
Comment (mgedmin) [inline src/z3c/relationfield/event.py]: Please use `"""` for docstrings.

File: src/z3c/relationfield/interfaces.py [hunk 2]
```diff
@@ -101,3 +103,6 @@ def convert():
 
         Returns real relation object
         """
+
+class IRelationBrokenEvent(IObjectModifiedEvent):
+    pass
```
Comment (mgedmin) [inline src/z3c/relationfield/interfaces.py]: This needs a docstring.

(Usually it's better to document the interface and then have the implementation's docstring refer to it.)

**Review by mgedmin — APPROVED** [review]
LGTM

**Comment by icemac** [comment]
@vangheem Is this PR ready to merge?

**Comment by vangheem** [comment]
yup

**Pull request merged**